modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,474 @@
+| Acknowledgement Mode a| Enumeration, <<SubscriberAckMode>>:
+
+** IMMEDIATE
+** AUTO
+** MANUAL |  The Acknowledgment mode to use for the messages retrieved by the subscribers. This affects only the Subscriber source and is unrelated to the consume operation. Refer to the <<SubscriberAckMode>> for more details on each strategy. |  AUTO |
+| Acknowledgement Timeout a| Number |  States the duration that a message is held by a broker waiting for an Acknowledgment or Not Acknowledgment. If neither an ACK or a NACK occurs before the timeout is exceeded, then the message is automatically returned to the destination by the broker, making it again available to any subscriber for redelivery. By default, this timeout is decided by the Subscriber depending on which strategy is used. When doing polling, then the ACK timeout is the one configured in the queue. When using the Prefetch strategy, messages are kept in a local buffer until they are dispatched to the flow. You can update the time to live (TTL) to avoid having messages be evicted. By default, the messages are fetched with a TTL of at least 2 minutes. |  0 |
+| Polling Time a| Number |  When prefetch is disabled, the polling time defines how often the subscriber should check for new messages. By default, this checks for messages every one second (only if the prefetch is disabled). The value of this parameter is defined in milliseconds. |  1000 |
+| Max Redelivery a| Number |  Number of redeliveries to accept for any given message. When the message redelivery count is higher than this threshold, then the message is ignored by the subscriber and not dispatched to the flow. For the message to be removed from the queue, the queue should be configured with a dead letter queue from the admin console. |  -1 |
+| Fetch Size a| Number |  Max number of messages to receive on each request. The response can contain fewer messages than this number depending on the load of the target queue. This number also affects the local buffer of the prefetched messages. Setting this fetchSize to zero  disables message prefetch. The prefetch buffer has a target size of three times the fetchSize, meaning that requests to retrieve messages continue as long as the local buffer is not full. Only in the case of a slow consumer, where the Mule Runtime has no more threads available to process the request, is the buffer limit going to be reached. Enable prefetch for maximum throughput, in cases where high performance on message delivery is required. In cases where controlled message retrieval is required, you can disable prefetch and use the fixed polling configuration. By default this tries to fetch the max number of 10 messages per request. |  10 |
```

Comment:
GLOBALLY attempt to reduce all Descriptions to the minimum required to define, as a reference, the named element. Omit conceptual information that belongs in a separate, concept topic. Never include instructions.

Reply:
Max > Maximum

Comma before "depending" (GLOBAL in this type of sentence structure).

END OF EDIT FOR 2.x files.